requirements.txt

```diff
@@ -1 +1,3 @@
 pytest==3.1.1
+coverage==4.4.1
+pytest-cov==2.5.1
```

Comment:
@extsoft But you said that we have to measure coverage each time we run unit tests.
command `py.test --cov=demo_python_at` runs all tests inside our projects with coverage.
Do I have to return back to version with installed library check?
```
if command coverage >/dev/null 2>&1 ; then
    py.test --cov=demo_python_at
else 
    py.test tests
fi
```